docs/guide/migrating-to-esm.md

```diff
@@ -1,20 +1,28 @@
 # Migrating to ESM in ApostropheCMS
 
-Starting with version 4.9.0, ApostropheCMS supports ECMAScript Modules (ESM). This guide will help you convert your existing project to use ESM.
+Starting with version 4.9.0, ApostropheCMS supports ECMAScript Modules (ESM). This guide will help you convert your existing project to use ESM and understand why this transition is valuable for your development workflow.
```

Comment:
Info box: "if you're starting a new project, good news! You don't have to sweat this stuff. Just fork [named branch] of each starter kit..."